lib/getLatestTips.ts

```diff
@@ -11,17 +11,32 @@ export const getLatestTips = async (): Promise<NextResponse> => {
     .from('tips_activity_log')
     .select('*')
     .order('created_at', { ascending: false })
-    .limit(11);
+    .limit(100);
+
 
   if (error) {
     return NextResponse.json(
       { message: 'Error fetching latest tips', error: error.message },
       { status: 500 },
     );
   }
+  const LIMIT = 11;
+  const MAX_SENDER_APPEARANCE = 2;
```

Comment:
can you please change this to `1`?
```suggestion
  const MAX_SENDER_APPEARANCE = 1;
```